execution-environments/darwin-base/Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM --platform=linux/amd64 rocker/rstudio:4.2
```

Comment:
Images should be kept small and I don't see why whole rstudio should be there. 
Can we change this to some image that only contains R runtime?

Reply:
yes

Reply:
The use case for including RStudio is to provide debugging tools but I agree in the ideal case (hopefully 90% of the time) there is no need for RStudio.